spec/bolt/transport/ssh_spec.rb

```diff
@@ -430,7 +432,8 @@ def result_value(stdout = nil, stderr = nil, exit_code = 0)
 
     context "as non-root" do
       let(:config) {
-        mk_config(host_key_check: false, sudo_password: password, run_as: user, user: user, password: password)
+        mk_config('host-key-check' => false, 'sudo-password' => password, 'run_as' => user,
+                  user: user, password: password)
```

Comment:
do 'user' and password also need to be 'user' => user ?

Reply:
I pulled some shenanigans in mk_config to stringify keys. I could change them all if that's confusing though.

Reply:
Eh, I don't care too strongly. Just wasn't sure if it would accept both.